src/miniscript/satisfy.rs

```diff
@@ -52,6 +52,9 @@ pub trait Satisfier<Pk: MiniscriptKey> {
     }
 
     /// Given a keyhash, look up the signature and the associated key
+    /// Even if signatures for public key Hashes are not available, the users
+    /// can use this map to provide pkh -> pk mapping which can use useful
+    /// for dissatisfying pkh.
```

Comment:
We could also create a separate function for this mapping.

Reply:
Yeah, we should. I don't understand how users can use the current function to look up public keys?